src/repositories/application-repository.js

```diff
@@ -434,3 +432,74 @@ export const updateEligiblePiiRedaction = async (
   //   })
   // }
 }
+
+export const getApplicationsBySbi = async (db, sbi) => {
```

Comment:
the answer may well be yes. But do we need all of the following variations:

getBySbi
getApplicationsBySbi
getLatestApplicationBySbi
getApplicationRefencesBySbi

Can these be consolidated down a bit, or maybe if we really need them, have a couple of them act as wrappers around the others?